service/src/main/java/bio/terra/pipelines/notifications/TeaspoonsJobSucceededNotification.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.pipelines.notifications;
+
+import lombok.Getter;
+
+@Getter
+public class TeaspoonsJobSucceededNotification extends BaseTeaspoonsJobNotification {
+  private static final String NOTIFICATION_TYPE = "TeaspoonsJobSucceededNotification";
+  public final String notificationType;
+  public final String quotaConsumedByJob;
```

Comment:
these two fields seem to be in both subclasses of the base class.  Any reason to not put them in the base class?